vmdb/app/models/vm_or_template.rb

```diff
@@ -1064,7 +1064,7 @@ def storage2active_proxies(all_proxy_list = nil)
     # MiqServer coresident proxy needs to contact the host and provide credentials.
     # Remove any MiqServer instances if we do not have credentials
     rsc = self.scan_via_ems? ? self.ext_management_system : self.host
-    proxies.delete_if {|p| MiqServer === p} if rsc && rsc.authentication_invalid?
+    proxies.delete_if {|p| MiqServer === p} if rsc && rsc.missing_credentials?
```

Comment:
@jrafanie As luck would have it we do have logic in place to notify users that proxies were eliminated.  The code could use some updates but the mechanism is in place. https://github.com/ManageIQ/manageiq/blob/master/vmdb/app/models/vm_or_template.rb#L980